.prettierrc

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Config file cannot be empty, we need to set some rules to be applied in the codebase. Let's follow this [tutorial](https://www.youtube.com/watch?v=TYxEydiqazc)